Packs/AzureSQLManagement/Integrations/AzureSQLManagement/AzureSQLManagement.yml

```diff
@@ -0,0 +1,593 @@
+category: IT Services
+commonfields:
+  id: Azure SQL Management
+  version: -1
+configuration:
+- display: Application ID
+  hidden: false
+  name: app_id
+  required: true
+  type: 0
+- display: Subscription ID
+  hidden: false
+  name: subscription_id
+  required: true
+  type: 0
+- display: Resource Group Name
+  hidden: false
+  name: resource_group_name
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Microsoft Azure SQL Database is a managed cloud database provided as
+  part of Microsoft Azure.
+display: Azure SQL Management (Beta)
+name: Azure SQL Management
+script:
+  commands:
+  - deprecated: false
+    description: Run this command to start the authorization process and follow the
+      instructions in the command results.
+    execution: false
+    name: azure-sql-auth-start
+  - deprecated: false
+    description: Run this command to complete the authorization process. Should be
+      used after running the azure-sql-auth-start command.
+    execution: false
+    name: azure-sql-auth-complete
+  - deprecated: false
+    description: Run this command if for some reason you need to rerun the authentication
+      process.
+    execution: false
+    name: azure-sql-auth-reset
+  - deprecated: false
+    description: Tests the connectivity to the Azure SQL Management.
+    execution: false
+    name: azure-sql-auth-test
+  - arguments:
+    - default: false
+      description: The maximum number of servers returned to the War Room. Default
```

Comment:
no need to mention default values in the description, it is added automatically by the platform with the specified value in the `defaultValue` property